src/datepicker/datepicker.js

```diff
@@ -458,11 +461,13 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
 }])
 
 .controller('UibMonthpickerController', ['$scope', '$element', 'dateFilter', function(scope, $element, dateFilter) {
+  var columns;
   this.step = { years: 1 };
   this.element = $element;
 
   this.init = function(ctrl) {
     angular.extend(ctrl, this);
+    columns = ctrl.monthColumns;
```

Comment:
Why does this need to be stored - couldn't one just use `ctrl.monthColumns` when needed?